molecule/provisioner/ansible/plugins/libraries/molecule_vagrant.py

```diff
@@ -354,6 +354,15 @@ def destroy(self):
 
         self._module.exit_json(changed=changed)
 
+    def halt(self):
+        changed = False
+        cd = self._created()
+        if cd:
```

Comment:
Why not "if self._created()"?

Reply:
Thats a good observation @berendt ... I'd like to keep my PR pretty lean though and since that paradigm is already used throughout the file. I think filing a different PR to cleanup that specific logic would be a better course of action.

Reply:
@msheiny Have not checked the rest of the file. if it is used everywhere else it fits.

Reply:
@msheiny good catch, cleaned those up (https://github.com/metacloud/molecule/commit/507893079802ca594a671bafb90c21f9255f1387)